src/server/handlers/github-auth.js

```diff
@@ -0,0 +1,59 @@
+import request from 'request';
+import logging from '../logging';
+
+const AUTH_SCOPE = 'admin:repo_hook';
```

Comment:
This level of scope works for adding webhooks, we'll probably need other scopes for accessing private repos.